app/jobs/govbox/sync_box_job.rb

```diff
@@ -7,13 +7,17 @@ def perform(box, upvs_client: UpvsEnvironment.upvs_client)
       edesk_api = upvs_client.api(box).edesk
       response_status, raw_folders = edesk_api.fetch_folders
 
+      sync_exclude_folder_ids = box.settings["sync_exclude_folder_ids"]
```

Comment:
```suggestion
      sync_exclude_folder_ids = box.settings["sync_exclude_folder_ids"] || []
```

A potom dole uz nemusis robit s `&`